packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -220,14 +219,14 @@ bug_report=--bug-report
 bug_report=
 use_booster=--use-booster
 # use_booster=
+state_diff="./snapshots/state-diff/Kontrol-Deploy.json"
 
 #########################################
 # List of tests to symbolically execute #
 #########################################
 tests=""
-tests+="--match-test CounterTest.test_SetNumber "
 #tests+="--match-test OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused "
```

Comment:
Why don't we also run this proof?

Reply:
Because in this PR, we're not adding the symbolic support for `bytes` and `bytes[],` hence we should use the workarounds if we want to run `prove_proveWithdrawalTransaction_paused.` But because of the `bytes[]` argument, it is too slow even with the fast summarization in place to be run in CI